### DISCUSSION TITLE
Updating Fooocus with "RealisticstockPhotos20.safetensors"

### DISCUSSION THREAD

**Discussion by TPK411** [lllyasviel/Fooocus#2227] [Q&A] [answered]
Hey. Noob question sorry: how do I add the file I've downlaoded to update my Fooocus to the new models? 

When booting up it told me i'm running old models, so I've donwloaded these but don't know how to integrate!

Thanks for your time and help :)

B

**Comment by mashb1t** [comment, marked as answer]
You can put the new model in models > checkpoints and Fooocus will auto-detect it when either reloading the models using the reload models button or when restarting.

**Reply by TPK411** [reply]
thank you!
